binance-smart-chain-rpc/src/constructors/ethereum/compiler.js

```diff
@@ -1,154 +1,184 @@
-const path = require('path')
```

Comment:
We want to separate PRs for different features, so this is best to go in a different commit and PR. We'll review them together this time.

Reply:
Sure, I will make a separate PR from my forked repo to the master . I was working parallelly on bsc and api-keys , that's why it came with branch commit. I will do a separate PR once I finish the bsc setup for token launch . We may keep the bsc master and api-keys branch seperately. Once finishing this , I will do the changes in api-key branch